app/src/features/Batches/BatchCreate.tsx

```diff
@@ -1,61 +1,21 @@
 import React, { useState } from "react";
```

Comment:
since there's no business logic here, maybe we can move this file out of the feature folder?

Reply:
I moved it in the common/component folder !